### ISSUE TITLE
postCreateCommand fails after DevContainer Building

### ISSUE THREAD

**Issue by kse3hi** [eclipse-velocitas/velocitas-docs#100]
```
Running the postCreateCommand from devcontainer.json...

[178281 ms] Start: Run in container: /bin/sh -c .devcontainer/scripts/setup.sh
/bin/sh: 1: .devcontainer/scripts/setup.sh: not found
[182260 ms] Stop (3979 ms): Run in container: /bin/sh -c .devcontainer/scripts/setup.sh
[182262 ms] postCreateCommand failed with exit code 127. Skipping any further user-provided commands.
Done. Press any key to close the terminal.
```

**Comment by MP91** [comment]
@kse3hi Can you provide some more information to reproduce this?

I did a quick check with the current main branch and everything is working as expected.

**Comment by kse3hi** [comment]
Issue was fixed in PR#99
